packages/synapse-react-client/src/lib/containers/row_renderers/utils/ShowMore.tsx

```diff
@@ -7,6 +7,7 @@ type ShowMoreState = {
 
 type ShowMoreProps = {
   summary: string
+  maxCharacterCount?: number
```

Comment:
Introduced an optional maxCharacterCount parameter, defaults to the previous value